examples/example-gateway/endpoints/tchannel/baz/baz_call.go

```diff
@@ -59,6 +60,9 @@ func (w Workflow) Handle(
 	clientReqHeaders := make(map[string]string)
 	// passing endpoint reqHeaders to downstream client
 	for _, k := range reqHeaders.Keys() {
+		if textproto.CanonicalMIMEHeaderKey(k) == "X-Token" {
```

Comment:
Where does this `X-Token` value come from?

Reply:
upstream, i.e. endpoint request